redeem/path_planner/PathPlannerNative_wrap.h

```diff
@@ -0,0 +1,60 @@
+/* ----------------------------------------------------------------------------
```

Comment:
This file is generated during the build and should not be checked in. Possibly even add it to gitignore.